build/includes/sdk.mk

```diff
@@ -183,7 +183,7 @@ run-sdk-conformance-test-rest:
 	$(MAKE) run-sdk-command COMMAND=clean SDK_FOLDER=restapi
 
 # Run a conformance test for all SDKs supported
-run-sdk-conformance-tests: run-sdk-conformance-test-node run-sdk-conformance-test-go run-sdk-conformance-test-rust run-sdk-conformance-test-rest run-sdk-conformance-test-cpp
+run-sdk-conformance-tests: run-sdk-conformance-test-rust run-sdk-conformance-test-node run-sdk-conformance-test-go run-sdk-conformance-test-rest run-sdk-conformance-test-cpp
```

Comment:
I am noting that the order of items here changed 🤔 this _shouldn't_ break anything, but maybe we have an order dependency I'm just not seeing?

Reply:
I just added it back. Removed before to make sure it's the `run-sdk-conformance-test-rust` causes the issue.

Reply:
Okay, after adding back the `run-sdk-conformance-test-rust`, the test starts to fail again...😭

Reply:
..what if you put it back  to the same order it was in previously? Does that make any difference?

Reply:
Cloud build does run all of them in parallel, so in theory it shouldn't matter - but now I'm wondering?

https://github.com/googleforgames/agones/blob/84277fb04d9915410e77fb22096fed503d9a4ffb/cloudbuild.yaml#L259

Reply:
I didn't touch this file till an hour ago and it failed more than 5 times already, but I can definitely change it back.

Reply:
😞 yeah, I'm not sure what is happening here, just removing all the possible issues I can think of.

🤞🏻 hopefully this will lead to finding whatever is flaky about the sdk tests that we've never been able to work out.

Reply:
I have tried to remove the `run-sdk-conformance-test-rust` and it works, now I will try to only run it without other sdks and see... The error message showing transport failure is so vague...